### ISSUE TITLE
Query on PE marking

### ISSUE THREAD

**Issue by Li-Zizhen** [nus-cs2103-AY2324S2/forum#974]
- If the teaching team agrees that it is a bug, but the severity I labeled is wrong, will this bug be counted as one of the four?
![image](https://github.com/nus-cs2103-AY2324S2/forum/assets/122284239/d719d2b8-2bb2-4db3-b104-41e536e8eb4f)

- If the bug severity/type I label is wrong, will I earn credit for reporting this bug?

**Comment by damithc** [comment]
> * If the teaching team agrees that it is a bug, but the severity I labeled is wrong, will this bug be counted as one of the four?
>   ![image](https://private-user-images.githubusercontent.com/122284239/324750575-d719d2b8-2bb2-4db3-b104-41e536e8eb4f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM4NjgxMzEsIm5iZiI6MTcxMzg2NzgzMSwicGF0aCI6Ii8xMjIyODQyMzkvMzI0NzUwNTc1LWQ3MTlkMmI4LTJiYjItNGRiMy1iMTA0LTQxZTUzNmU4ZWI0Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyM1QxMDIzNTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iYjVkZDg5Y2UyNjVmN2NlOTg1NTUyY2MxMTIxNjZiZmM4ZmZjOTdkMDY2ZDAzZjJhOTZmNTU4YjI1YWFjYTYyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.v3-2unqz8U8ZbLL3yqZ-24PDg_JTJYVb3jZS597030s)

@Li-Zizhen Yes. All credit earning bugs count. By the way, 4 is an indicative number only. The real bar also factors in the credit earned by each bug, not just a simple count of bugs.


> * If the bug severity/type I label is wrong, will I earn credit for reporting this bug?

Yes. You still earn credit even if the labels were wrong, provided the bug is accepted or at least NotInScope.
